src/Components/LandingPage.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 function LandingPage () {
     return (
         <div>
             <h1>Welcome to Litter.ai</h1>
             <p>With just a click, sort your garbage and save the world</p>
+            <div style={{ position: 'fixed', bottom: '20px', left: '50%', transform: 'translateX(-50%)', width: '200px' }}>
```

Comment:
lets make a css file and this should also be in its own directiory. src/components/landingPage/LandingPage.js and another file in there LandingPage.css that the js file imports. That is where all styles for this component should go.

Reply:
was the styling added?

Reply:
I didn't add any styling. I'm going to add the styling on a separate PR